### ISSUE TITLE
Events page: Add cost as a filter

### ISSUE THREAD

**Issue by mandily-p** [BCcampus/eypd#550]
As a learner,
I want to be able to find the free courses as easily as possible,
Because I don't want to pay or waste my valuable time.

### Description 
During user testing, it was mentioned that some learner users may only be interested in free courses to get their hours. We should cater to that audience and add a "cost" column to the table and allow users to filter by free or paid.

**Comment by kathreenriel** [comment]
This is related to recent discussions with WestCoast CCRR needing a category for Vancouver Early Years Partnership

**Comment by mandily-p** [comment]
I shortened up the "Search by category" field and added the "cost" field beside it. Same size as the date fields, but it doesn't have to be that way. Whatever works with the grid.

I purposely didn't do filters .01 - 19.99, 20 - 29.99, 30 - 39.99 etc
I figured people are filtering by cost because it's what they can afford, not that they wouldn't want to pay less than $50. 
So each consecutive option would include the events that were in the previous option.

![screen shot 2018-11-05 at 1 28 50 pm](https://user-images.githubusercontent.com/20425828/48027866-d1bdec00-e0fe-11e8-8d3d-e43d58721bef.png)
![screen shot 2018-11-05 at 1 29 04 pm](https://user-images.githubusercontent.com/20425828/48027869-d387af80-e0fe-11e8-8a7b-3d960e47bf7b.png)
